modules/cudf/src/series.ts

```diff
@@ -906,14 +907,44 @@ export class AbstractSeries<T extends DataType = any> {
   }
 
   /**
-   * Removes duplicate values from the Series.
+   * Returns a new Series with only the unique values that were found in the original
    *
    * @param nullsEqual Determines whether nulls are handled as equal values.
    * @param memoryResource Memory resource used to allocate the result Column's device memory.
    * @returns series without duplicate values
+   *
+   * @example
+   * ```typescript
+   * import {Series} from '@rapidsai/cudf';
+   *
+   * // Float64Series
+   * Series.new([null, null, 1, 2, 3, 3]).unique(true) // [null, 1, 2, 3]
+   * Series.new([null, null, 1, 2, 3, 3]).unique(false) // [null, null, 1, 2, 3]
+   * ```
    */
   unique(nullsEqual = true, memoryResource?: MemoryResource) {
-    return this.__construct(this._col.drop_duplicates(nullsEqual, memoryResource));
+    return this.dropDuplicates([0], 'keep_first', nullsEqual, true, memoryResource);
+  }
+
+  /**
+   * Returns a new Series with duplicate values from the original removed
+   *
+   * @param keys The indices of columns that should be considered when searching for duplicate
+   *   values
+   * @param keep Determines whether to keep the first, last, or none of the duplicate items.
+   * @param nullsEqual Determines whether nulls are handled as equal values.
+   * @param nullsFirst Determines whether null values are inserted before or after non-null values
+   * @param memoryResource Memory resource used to allocate the result Column's device memory.
+   * @returns series without duplicate values
+   */
+  dropDuplicates(keys: number[],
+                 keep: keyof typeof DuplicateKeepOption,
+                 nullsEqual: boolean,
+                 nullsFirst: boolean,
+                 memoryResource?: MemoryResource) {
+    return Series.new(new Table({columns: [this._col]})
+                        .dropDuplicates(keys, keep, nullsEqual, nullsFirst, memoryResource)
```

Comment:
I think `keys` will always be `[0]` for `Series`, so I'm wondering if I should remove the `keys` argument from this method and just hardcode `[0]` in the `Table.dropDuplicates` call.

Reply:
It will be for Series, but not for DataFrame. `Series.dropDuplicates` should call the `Table.dropDuplicates()` with `[0]` as the keys.